UserRelated/ExampleData/createExampData.py

```diff
@@ -0,0 +1,31 @@
+from OsmDataCollector import OsmDataCollector
+import json
+import os
+import shutil
+
+baiersbronn = [8.1584, 48.4688, 8.4797, 48.6291]
+COORS_PATH = 'tracks_gps_points\\'
+
+
+def create_osm_db(area, n=10):
+    """
+    Creates a json with osm tracks data, for testing purpose.
+    :param area: the bounding box of the tracks to be generated.
+    :param n: The number of fake osm recodes to be generated.
+    """
+    if os.path.isdir(COORS_PATH):
+        shutil.rmtree(COORS_PATH)
+    os.makedirs(COORS_PATH)
+
+    osm_data = OsmDataCollector(area)
+    data = {'tracks': []}
+    for i in range(n):
+        if len(osm_data.tracks) > i:
```

Comment:
maybe we should save to file more often to prevent the data proccessing from getting lost while crawling due to exceptions?
i think you should start testing it (without the similarity part because it's not complete yet)  in order to check 
the flow of the mining process: and the way it handles exeptions. i had some troubles with it. if it works out- then great! it you encounter some troubles we can discuss it in order to save some time and effort.

Reply:
I generally thinks it's a good idea, but that it something we can do in the future and shouldn't interfere with the current merge. I am adding this to the project's backlog and resolving this issue.